# ContactTest
>解决不能推送到GitHub.  
rejected Updates were rejected because the remote contains work that you do not have locally. This is usually caused by another repository pushing to the same ref. You may want to first integrate the remote changes before pushing again.

__git pull origin master --allow-unrelated-histories__

### 主要功能

>添加联系人，编辑联系人信息，删除联系人，搜索联系人

### 演示视频

[传送门](https://github.com/kaitiandeng/ContactTest/blob/master/20180531_191021.mp4)

### 动态图 

* 不是很清晰


